src/SmartComponents/SubscriptionsUtilized/SubscriptionsUtilizedCard.js

```diff
@@ -1,32 +1,138 @@
 
 /**
  * Subscriptions utilized card for showing the portion of Subscriptions used.
  */
-const SubscriptionsUtilizedCard = ({ intl }) => {
-    return <TemplateCard appName='SubscriptionsUtilized'>
-        <TemplateCardHeader subtitle={ intl.formatMessage(messages.subscriptionsUtilized) }/>
-        <TemplateCardBody>
-            <ProgressTemplate
-                title="Red Hat JBoss"
-                value="0"
-            />
-            <ProgressTemplate
-                title="Red Hat Openshift"
-                value="0"
-                variant="danger"
-            />
-        </TemplateCardBody>
-    </TemplateCard>;
+const SubscriptionsUtilizedCard = ({ intl, subscriptionsUtilizedProductOne, subscriptionsUtilizedProductOneFetch,
+    subscriptionsUtilizedProductOneFetchStatus, subscriptionsUtilizedProductTwo, subscriptionsUtilizedProductTwoFetch,
+    subscriptionsUtilizedProductTwoFetchStatus }) => {
+
+    const [products, setProducts] = useState([]);
+
+    useEffect(() => {
+        const { startDate, endDate } = setRangedDateTime();
+        const options = {
+            granularity: RHSM_API_QUERY_GRANULARITY_TYPES.DAILY,
+            beginning: startDate.toISOString(),
+            ending: endDate.toISOString()
+        };
+
+        subscriptionsUtilizedProductOneFetch(RHSM_API_PRODUCT_ID_TYPES.OPENSHIFT, options);
+        subscriptionsUtilizedProductTwoFetch(RHSM_API_PRODUCT_ID_TYPES.RHEL, options);
+    }, [
+        subscriptionsUtilizedProductOneFetch,
+        subscriptionsUtilizedProductTwoFetch
+    ]);
+
+    useEffect(() => {
+        const chartData = { productOne: {}, productTwo: {} };
+
+        if (subscriptionsUtilizedProductOneFetchStatus === 'fulfilled' || subscriptionsUtilizedProductTwoFetchStatus === 'fulfilled') {
+            const [productOneReport = {}, productOneCapacity = {}] = Immutable.asMutable(subscriptionsUtilizedProductOne, { deep: true }) || [];
+            const [productTwoReport = {}, productTwoCapacity = {}] = Immutable.asMutable(subscriptionsUtilizedProductTwo, { deep: true }) || [];
+
+            chartData.productOne = filterChartData(
+                productOneReport[RHSM_API_RESPONSE_DATA],
+                productOneCapacity[RHSM_API_RESPONSE_DATA],
+                [RHSM_API_RESPONSE_DATA_TYPES.SOCKETS]
+            );
+
+            chartData.productTwo = filterChartData(
+                productTwoReport[RHSM_API_RESPONSE_DATA],
+                productTwoCapacity[RHSM_API_RESPONSE_DATA],
+                [RHSM_API_RESPONSE_DATA_TYPES.CORES]
+            );
+        }
+
+        setProducts(chartData);
+    }, [
+        subscriptionsUtilizedProductOne,
+        subscriptionsUtilizedProductTwo,
+        subscriptionsUtilizedProductOneFetchStatus,
+        subscriptionsUtilizedProductTwoFetchStatus
+    ]);
+
+    const { productOne = {}, productTwo = {} } = products;
+
+    const productTwoTooltip = (
+        <ul>
+            <li>RHEL sockets: {productTwo.report}</li>
```

Comment:
Have to insist on translations here (and everywhere else), s not the kinda tech debt we're okie dokie accumulating 😟